front-dispatcher/src/features/SearchBar/SearchBar.tsx

```diff
@@ -18,25 +18,37 @@ interface searchbarProps {
 }
 const SearchBar = ({ fullScreen, mobileBackFC }: searchbarProps) => {
   const [focused, setFocused] = useState<boolean>(false);
+  const searchDivRef = useRef<HTMLDivElement>(null);
   const filterState = useAppSelector((state) => state.filters);
   const { endpoint } = filterState;
+  useEffect(() => {
+    const handleClickOutside = (event: Event) => {
+      if (
+        searchDivRef.current &&
+        !searchDivRef.current.contains(event.target as Node)
```

Comment:
Can you explain me the meaning of event.target as Node ?

Reply:
so typescript understands e.target is an element